contrib/sarplus/azure-pipelines.yml

```diff
@@ -0,0 +1,93 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+pr:
+  branches:
+    include:
+      - staging
+      - master
+  paths:
+    include:
+      - contrib/sarplus/*
+
+# no CI trigger
+trigger: none
+
+jobs:
+- job: 'Test'
+  pool:
+    vmImage: 'Ubuntu 16.04'

```

Comment:
Do we want to reuse the agent pool for other builds?

Reply:
aren't those the multi-tenant agents anyway?

Reply:
Not sure but this will create new agent pool for VM to run the build or it will use the existing one? I am thinking of cost, but this may be cheaper because it does not have to use the GPU based VM agent we have in place.